extensions/opentelemetry/runtime/src/main/java/io/quarkus/opentelemetry/runtime/tracing/intrumentation/vertx/HttpInstrumenterVertxTracer.java

```diff
@@ -256,6 +249,14 @@ public void onStart(
                 if (clientIp != null) {
                     attributes.put(HTTP_CLIENT_IP, clientIp);
                 }
+                String remoteHostname = VertxUtil.extractRemoteHostname((HttpServerRequest) httpRequest);
```

Comment:
There is now an AttributesExtractor for these attributes. Check `NetServerAttributesGetter`. We can use that instead.

Reply:
I know, but because its implementation should get data from Netty, ideally.
I prefer to keep things like they are for now because this data comes mostly from the host header and the Netty data might not align.
What do you think?

Reply:
Why should it get it from Netty? It is the exact same code you have to get the data, just on a different extractor. For instance: https://github.com/smallrye/smallrye-opentelemetry/blob/1e2db800ff0bd73d7ca3080b0c354f88edbd6f96/implementation/rest/src/main/java/io/smallrye/opentelemetry/implementation/rest/OpenTelemetryServerFilter.java#L116-L131

Reply:
ok, will move the impl to NetServerAttributesGetter